### PR TITLE
add: `bottles`, `python3-fvs`, and `python3-vkbasalt-cli` 

### PR DESCRIPTION
**Pull request by AozoraDev**

### CODE REVIEW

File: packages/bottles/bottles.pacscript [hunk 1]
```diff
@@ -0,0 +1,84 @@
+pkgname="bottles"
+pkgver="51.13"
+pkgdesc="Easily manage wine and proton prefix"
```
Comment (vigress8) [inline packages/bottles/bottles.pacscript:3]:
```suggestion
pkgdesc="Easily manage wine and proton prefixes"
```

File: packages/bottles/bottles.pacscript [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  "ubuntu:oracular"
+  "ubuntu:devel"
+)
+source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/refs/tags/${pkgver}.tar.gz")
+sha256sums=("ba582ef2eaec4f3e533fd92b9b064223373e55b4d942113f071fc90800d49c34")
+
+build() {
+  if [[ -d Bottles ]]; then
+    rm -rf Bottles
+  fi
```
Comment (vigress8) [inline packages/bottles/bottles.pacscript:74]: Why is this needed?

**Review by vigress8 — CHANGES_REQUESTED** [review]

File: packages/bottles/bottles.pacscript (hunk 2 above)
Comment (AozoraDev) [reply packages/bottles/bottles.pacscript:74]: idk.
Just leave it there, because *if it work, it work*.

**Review by vigress8 — APPROVED** [review]
